### ISSUE TITLE
[Proposal] Please ddd support for fine-tuned models

### ISSUE THREAD

**Issue by jasonlim131** [neelnanda-io/TransformerLens#535]
### Proposal 

Please add support for finetuned-variants of pretrained models, like Pythia < 1B and gpt2-small.

### Motivation

I'm frustrated when I find a very appropriate dataset and circuit I would like to investigate using TransformerLens but the library does not support finetuned models. 

### Pitch

A clear and concise description of what you want to happen. It is useful if you relate proposals to existing features or lackthereof as well as relevant mechanistic interpretability techniques or model architectures.

### Alternatives

For fine-tuned gpt-2 small and other models, use classical linear probing, train an attention SAE, or try implementing activation patching from scratch.

### Additional context

Finetuning is required because I suspect that gpt-2 small does not solve the tasks of interests. I will confirm this, but I'm pretty sure. 

### Checklist

- [O] I have checked that there is no similar [issue](https://github.com/neelnanda-io/transformerlens/issues) in the repo

**Comment by neelnanda-io** [comment]
Can you be a bit more specific about your use case? Are you fine tuning the
models yourself, or are you using someone else's? Because you can just load
gpt2 into TransformerLens and fine tune to your heart's content, this is
all supported fine. It's fundamentally the same model, just different
weights. You can also save those tuned weights, load gpt2 normally, and
then do model.load_and_process_state_dict

If you want to take a fine tuned model from HuggingFace that also works,
but you'll need to use the weight converters in loading_from_pretrained to
get it into TL format

**Comment by jasonlim131** [comment]
yeah, I'm fine-tuning it to solve LogicQA deduction tasks. I just wasn't sure if there existed a function to load into TransformerLens _after_ finetuning, but I think the first option works. Thanks for the clarification.
